plugins/plugin-chart-echarts/src/Timeseries/Area/controlPanel.tsx

```diff
@@ -251,6 +252,18 @@ const config: ControlPanelConfig = {
             },
           },
         ],
+        [
+          {
+            name: 'show_value',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show Value'),
+              default: showValue,
+              renderTrigger: true,
+              description: t('Show the value on top of the chart'),
+            },
+          },
+        ],
```

Comment:
To avoid duplication, we should probably place this in `controls.ts` and then reuse it in all charts where it's needed

Reply:
> This is really great work @stephenLYZ! A few minor comments. Also, it would be great if we could add a checkbox for "Show only total" that is visible if `show_value` is enabled that toggles between either only showing the stacked total value or the individual values. So if we have the following series:
> 
> * bicycles: 100
> * cars: 200
> 
> In this case:
> 
> * If we `show_value` is unchecked, `only_total` control isn't displayed at all
> * If we only check `show_value` but not `only_total`, we would see 100 next to bicycles and 200 next to cars.
> * If we check both `sow_value` and `only_total`, we would see 300 next to whichever value is on top of the stack.

Interesting~ I had previously thought that total value was only for stack scenarios. 😂. I think I can put it for the next task.